webapp/src/pxtjson.tsx

```diff
@@ -85,7 +85,7 @@ export class Editor extends srceditor.Editor {
         return (
             <div className="ui content">
                 <div className="ui segment form text" style={{ backgroundColor: "white" }}>
-                    <sui.Input label={lf("Name")} value={c.name} onChange={setFileName}/>
+                    <sui.Input id={"fileNameInput"} label={lf("Name")} ariaLabel={lf("Type a name for your project")} value={c.name} onChange={setFileName}/>
                     {userConfigs.map(uc =>
```

Comment:
We use "Pick a name for your project" elsewhere? Why change that? Let's be consistent.